### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by matthewwong525** [fleetingnotes/fleeting-notes-flutter#929]
**Is your feature request related to a problem? Please describe.**
Support languages other than English. As english may not be everyones main language

**Describe the solution you'd like**
Use flutter localization library

**Comment by fallsdevil** [comment]
Yes! using crowdin or weblate I would be happy to help translate, it's a great idea.
